src/moments.jl

```diff
@@ -1,52 +1,100 @@
 ##### Weighted var & std
 
 ## var
-
 """
-    varm(x, wv::WeightVec, m, [dim])
+    varm(x, wv::AbstractWeights, m, [dim, corrected])
 
 Return the variance of a real-valued array `x` with a known mean `m`, optionally
-over a dimension `dim`. The weighting vector `wv` specifies frequency weights
-(also called case weights) for the result.
-
-This function differs from its counterpart in Base in that Bessel's correction
-is not used. That is, here the denominator for the variance is `sum(wv)`,
-whereas it's `length(x)-1` in `Base.varm`. The impact is that this is not a
-weighted estimate of the population variance based on the sample; it's the weighted
-variance of the sample.
+over a dimension `dim`. Observations in `x` or weighted via `wv`.
+
+In base julia a biased variance (`corrected=false`) is calculated as:
+
+``\\frac{1}{N} \sum\limits_{i = 1}^N {\left( {x_i - \bar x} \right)^2 }``
+
+An unbiased variance (`corrected=true`) is calculated by replacing
+``\\frac{1}{N - 1}`` with ``\\frac{1}{N - 1}``
+(i.e. [Bessel's correction](https://en.wikipedia.org/wiki/Bessel's_correction)).
+
+Here we calculate the biased weighted variance (`corrected=false`) as:
+
+``\\frac{1}{\sum{w}} \sum\limits_{i = 1}^N {w_i\left( {x_i - \bar x} \right)^2 }``
```

Comment:
Also, it looks like we already aren't using the extra backslash.

https://github.com/JuliaStats/StatsBase.jl/blob/master/src/scalarstats.jl#L45